lib/chef/knife/clc_base.rb

```diff
@@ -57,6 +57,14 @@ def parse_and_validate_parameters
           def execute
           end
 
+          def os_windows?
```

Comment:
These method names are a bit misleading. We have at least two machines (client where Knife CLC is running, and host). It's not obvious by a method name which machine platform it reports.